src/clover/__utils__.py

```diff
@@ -1906,6 +1906,27 @@ def from_dict(cls, hot_water_inputs: Dict[str, Any], logger: logging.Logger) ->
         )
 
 
+class GridType(enum.Enum):
```

Comment:
@paulharfouche , if you could fill these in the same way as the other classes in CLOVER, i.e., saying what the attributes are, and formatting them in the same way, that would be great. For the `enum.Enum` ones, take a look at the `BColours` class, and for the `@dataclasses.dataclass`, take a look at maybe `PVPanel` in the `solar.py` module, or anything in this file which has this __decorator__ :smile:

Reply:
We're here telling Python that this is a class, that can represent some thing, in this instance, it's representing the type of the grid. The `enum.Enum` class basically works kind of like a boolean, but, rather than being `True` or `False`, it can be `CURRENT_DRAW` or `DAILY_POWER`. No other values are allowed, other than what we define here, and so they can correspond to real-world choices between things that are enforced to be only a certain thing :smile: